### PR TITLE
Add registration deadline support

### PR DESCRIPTION
**Pull request by tannewt**

### CODE REVIEW

File: scripts/election.py [hunk 1]
```diff
@@ -154,6 +156,10 @@ def sort_key(x):
     "poll.in_person_by": "Last day to vote in person",
     "poll.early.in_person_by": "Last day to vote early in person",
     "poll.overseas_military.received_by": "Last day for election officials to receive military and overseas ballots"
```
Comment (askpatrickw) [inline scripts/election.py]: Missing a comma here

File: scripts/election.py [hunk 2]
```diff
@@ -190,15 +198,16 @@ def sort_key(x):
...
 
             mail = copy.deepcopy(reminder)
             mail["date"] = reminder["date"] - one_day - one_week
-            mail["name"] = f"Mail {item}! It must be received"
+            mail["name"] = f"Mail {item}"
             reminders.append(mail)
 
             post_office = copy.deepcopy(reminder)
-            post_office["name"] = f"Mail {item} at the post office! It must be received"
+            post_office["name"] = f"Mail {item} at the post office"
```
Comment (askpatrickw) [inline scripts/election.py:210]: I'm not completely following this. I assume someplace else the date and the "It must be received" are merged into a string on a calendar item and its not missing here.

**Review by askpatrickw — CHANGES_REQUESTED** [review]
Missing one comma... is all I see for sure. 

As an FYI for Colorado, there was a 'change party affiliation' date for the primary, but it was the same as the registration, so I figured updating your registration is the same as registration. BUT it could have been registration.change_affiliation.

File: scripts/election.py (hunk 2 above)
Comment (tannewt) [reply scripts/election.py:210]: Ya, it's moved to the "explanation" key. "name" needs to be a phrase so that it can be combined with other reminders.

**Comment by tannewt** [comment]
> As an FYI for Colorado, there was a 'change party affiliation' date for the primary, but it was the same as the registration, so I figured updating your registration is the same as registration. BUT it could have been registration.change_affiliation.

Ya, this happens other places too. We could sketch that out for now.

**Pull request merged**